src/page-modules/assistant/trip/trip-pattern/index.tsx

```diff
@@ -108,15 +118,27 @@ export default function TripPattern({
                     )}
                   </div>
 
-                  <Typo.span textType="body__tertiary">
-                    {formatToClock(leg.expectedStartTime, language, 'floor')}
-                  </Typo.span>
+                  <div className={style.timeStartContainer}>
+                    <Typo.span textType="body__tertiary">
+                      {formatToClock(leg.expectedStartTime, language, 'floor')}
+                    </Typo.span>
+                    {!isEqualOnMinuteLevel(
+                      leg.aimedStartTime,
+                      leg.expectedStartTime,
+                    ) && (
+                      <Typo.span
+                        textType="body__tertiary"
+                        className={style.lineThrough}
```

Comment:
Isn't there a `textType` with strike-through?

Reply:
Yes, but the font-size is of that text type is larger than  body__tertiarys font-size.

Reply:
That shouldn't be the case? 

`textType="body__tertiary"` and `textType="body__tertiary--strike"` should be the same size.

Reply:
ah, my mistake. I mixed body__primary--strike and body__tertiary--strike. You are right.

Reply:
Still need the color change though, right?

Reply:
True, added it back now.